src/api/v3/feedback/page_feedback.py

```diff
@@ -82,10 +94,12 @@ def feedback(request, region_slug, language_code):
     if not page:
         slug = feedback_data.permalink.split("/")[-1]
         potential_page_translation = PageTranslation.objects.get(
-            slug=slug,
-            language__code=language_code,
+            slug=slug, language__code=language_code, page__region=region
         )
-        if potential_page_translation.permalink == feedback_data.permalink:
+        feedback_data.permalink = urlparse(feedback_data.permalink).path
+        if potential_page_translation.permalink.strip(
+            "/"
+        ) == feedback_data.permalink.strip("/"):
```

Comment:
```suggestion
        if potential_page_translation.permalink == feedback_data.permalink.strip("/"):
```
The `potential_page_translation.permalink` cannot have a leading or trailing slash by definition... so no `strip()` necessary (which also allows the condition to be written in one line)